### ISSUE TITLE
package 'vscDebugger' not installed because it is not built for UCRT

### ISSUE THREAD

**Issue by Aliador** [ManuelHentschel/vscDebugger#182]
Error in unpackPkgZip(pkgs[i], pkgnames[i], lib, libs_only, lock, quiet) :
  package 'vscDebugger' not installed because it is not built for UCRT
Calls: install.packages -> .install.winbinary -> unpackPkgZip

**Comment by aragornii** [comment]
Any solution for this error? It's happening to me too in VS Code but I'm not sure to what is due.

**Comment by mvwestendorp** [comment]
A workaround is to build it from source using R 4.2 on Windows.
The error relates to the UCRT runtime that is used starting from version 4.2 to support native UTF-8 encoding for Windows.
This is my binary build from simply cloning the repo and building the binary packages via RStudio:
[vscDebugger_0.4.7.zip](https://github.com/ManuelHentschel/vscDebugger/files/9231784/vscDebugger_0.4.7.zip)
 I have not seen the error since.

**Comment by aragornii** [comment]
Thanks @mvwestendorp, it worked perfectly.

**Comment by Ljtyxhc** [comment]
thanks you are the real hero

**Comment by zhcxww** [comment]
@mvwestendorp  you are so cool！！！😎

**Comment by filods** [comment]
does anyone know what command to run in github codespaces

**Comment by eitsupi** [comment]
> does anyone know what command to run in github codespaces

I don't think this issue is related to Linux on Codespaces as it only concerns Windows.

You should be able to use the debugger-configured container using the template below.
https://github.com/rocker-org/devcontainer-templates/tree/main/src/r-ver
